client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -125,104 +122,60 @@ const InstructionsRenderer = ({ testResult, testPageUrl, at }) => {
-            if (typeof value === 'string') {
-                if (value === '.')
-                    content = (
-                        <>
-                            {content}
-                            {value}
-                        </>
-                    );
-                else
-                    content = content = (
-                        <>
-                            {content} {value}
-                        </>
-                    );
-            } else if ('href' in value) {
-                const { href, description } = value;
-                content = (
-                    <>
-                        {content} <a href={href}>{description}</a>
-                    </>
-                );
-            }
-        }
-        return content;
-    };
-
-    const parseListContent = (instructions = [], commandsContent = null) => {
-        return instructions.map((value, index) => {
-            if (typeof value === 'string')
-                return (
-                    <li key={nextId()}>
-                        {value}
-                        {commandsContent &&
-                            index === instructions.length - 1 && (
-                                <BulletList>{commandsContent}</BulletList>
-                            )}
-                    </li>
-                );
-            else if (Array.isArray(value))
-                return (
-                    <li key={nextId()}>
-                        {parseRichContent(value)}
-                        {commandsContent &&
-                            index === instructions.length - 1 && (
-                                <BulletList>{commandsContent}</BulletList>
-                            )}
-                    </li>
-                );
-        });
-    };
-
-    const allInstructions = [
-        ...instructionsContent.instructions.instructions.instructions,
-        ...instructionsContent.instructions.instructions.strongInstructions,
-        instructionsContent.instructions.instructions.commands.description
-    ];
-
-    const commands =
-        instructionsContent.instructions.instructions.commands.commands;
+    let allInstructions;
+    const isV2 = testFormatVersion === 2;
+
+    if (isV2) {
+        const commandSettingSpecified = renderableContent.commands.some(
+            ({ settings }) => settings && settings !== 'defaultMode'
+        );
+
+        allInstructions = [
+            renderableContent.target.at.raw
+                .defaultConfigurationInstructionsHTML,
+            `${supportJson.testPlanStrings.openExampleInstruction} ${renderableContent.target.setupScript.scriptDescription}.`,
+            `${renderableContent.instructions.instructions} ${
+                supportJson.testPlanStrings.commandListPreface
+            }${
+                commandSettingSpecified
+                    ? ` ${supportJson.testPlanStrings.commandListSettingsPreface}`
+                    : ''
+            }`
+        ].map(e => unescape(e));
```

Comment:
I have a hard time seeing where an array item starts and stops here. This is mainly due to the fact that our project uses the default printwidth in prettier but maybe this could benefit from a couple of intermediate variables just to make the array easier to read.